localnet/common.sh

```diff
@@ -102,7 +102,7 @@ build-localnet() {
   DOT_GENESIS_HASH=$(echo $REPLY | grep -o '\"result\":\"0x[^"]*' | grep -o '0x.*')
 
   echo "🐛 Fix solana symlink issue ..."
-  rm $SOLANA_BASE_PATH/test-ledger/snapshot/100/accounts_hardlinks/account_path_0
+  rm -f $SOLANA_BASE_PATH/test-ledger/snapshot/100/accounts_hardlinks/account_path_0
   ln -s $SOLANA_BASE_PATH/test-ledger/accounts/snapshot/100 $SOLANA_BASE_PATH/test-ledger/snapshot/100/accounts_hardlinks/account_path_0
```

Comment:
```suggestion
  ln -sf $SOLANA_BASE_PATH/test-ledger/accounts/snapshot/100 $SOLANA_BASE_PATH/test-ledger/snapshot/100/accounts_hardlinks/account_path_0
```
maybe let's apply force to `ln` command?